src/common/string_util.cpp

```diff
@@ -14,6 +14,40 @@
 
 namespace duckdb {
 
+const char *const StringUtil::BYTE_TO_HEXS = "000102030405060708090A0B0C0D0E0F"
```

Comment:
Using these large buffers seems a bit excessive for these functions. The reason we have such a large buffer for the string -> integer casting code is that (1) integer casting requires an expensive division for each lookup, and (2) this type of casting is very common so it is worth optimizing. In this case we are adding a large amount of binary code/code complexity for a likely less substantial speedup on a less frequently used function. 

Perhaps you could measure how much faster the code gets from using this larger buffer versus using a single regular buffer ("0123456789ABCDEF")?